docs/modules/control/move_to_a_pose_control/move_to_a_pose_control.rst

```diff
@@ -1,57 +1,109 @@
+
+- | **Kp_rho** : The linear velocity gain to translate the robot along a line towards the goal
+- | **Kp_alpha** : The angular velocity gain to rotate the robot towards the goal
+- | **Kp_beta** : The offset angular velocity gain accounting for smooth merging to the goal angle (i.e., it helps the robot heading to be parallel to the target angle.)
 
 
 Member function(s)
 ~~~~~~~~~~~~~~~~~~
 
 .. code-block:: ipython3
+
    calc_control_command(x_diff, y_diff, theta, theta_goal)
 
 Returns the control command for the linear and angular velocities as well as the distance to goal
 
 Parameters:
-- x_diff : The position of target with respect to current robot position in x direction
-- y_diff : The position of target with respect to current robot position in y direction
-- theta : The current heading angle of robot with respect to x axis
-- theta_goal : The target angle of robot with respect to x axis
+
+- | **x_diff** : The position of target with respect to current robot position in x direction
+- | **y_diff** : The position of target with respect to current robot position in y direction
+- | **theta** : The current heading angle of robot with respect to x axis
+- | **theta_goal** : The target angle of robot with respect to x axis
 
 Returns:
-- rho : The distance between the robot and the goal position
-- v : Command linear velocity
-- w : Command angular velocity
+
+- | **rho** : The distance between the robot and the goal position
+- | **v** : Command linear velocity
+- | **w** : Command angular velocity
+
+How does the Algorithm Work
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The distance between the robot and the goal position, :math:`\rho`, is computed as
+
+.. math::
+ \rho = \sqrt{(x_{robot} - x_{target})^2 + (y_{robot} - y_{target})^2}.
+
+The distance :math:`\rho` is used to determine the robot speed. The idea is to slow down the robot as it gets closer to the target.
+
+.. math::
+ v = K_P{_\rho} \times \rho\qquad (Eq. 1)
+
+Note that for your applications, you need to tune the speed gain, :math:`K_P{_\rho}` to a proper value.
+
+To turn the robot and align its heading, :math:`\theta`, toward the target position (not orientation),  :math:`\rho \vec{u}`, we need to compute the angle difference :math:`\alpha`. 
+
+.. math::
+ \alpha = (\arctan2(y_{diff}, x_{diff}) - \theta + \pi) mod (2\pi) - \pi
+
+The term :math:`mod(2\pi)` is used to map the angle to :math:`[-\pi, \pi)` range.
+
+Lastly to correct the orientation of the robot, we need to compute the orientation error, :math:`\beta`, of the robot.
+
+.. math::
+ \beta = (\theta_{goal} - \theta - \alpha + \pi) mod (2\pi) - \pi
+
+Note that to cancel out the effect of :math:`\alpha` when the robot is at the vicinity of the target, the term 
+
+:math:`-\alpha` is included.
+
+The final angular speed command is given by
+
+.. math::
+ \omega = K_P{_\alpha} \alpha - K_P{_\beta} \beta\qquad (Eq. 2)
+ 
```

Comment:
Maybe this space breaks the rendering:
<img width="685" alt="SS 2022-01-20 at 21 19 01" src="https://user-images.githubusercontent.com/3813847/150337421-e155811c-86de-4401-9d92-fc61791177d5.png">

```suggestion

```

Reply:
The leading space on line 97 was the issue. I fixed it.